.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  contents: write
+  issues: write
```

Comment:
Do we need to give this permission if we aren't checking issues?

Reply:
nope! deleting. if we do want to check issues in the future, we'll have to remember to add it back